src/develop/blend.c

```diff
@@ -2797,7 +3224,15 @@ int dt_develop_blend_process_cl(struct dt_iop_module_t *self, struct dt_dev_pixe
   const int gaussian = d->radius > 0.0f ? 1 : 0;
   const float radius = fabs(d->radius);
   const unsigned int mask_combine = d->mask_combine;
-  const int mask_display = piece->pipe->mask_display; /* we will transfer alpha channel of input if mask_display was activated by any _earlier_ module */
+
+  /* we will transfer alpha channel of input if mask_display was activated by any _earlier_ module */
+  const dt_dev_pixelpipe_display_mask_t mask_display = piece->pipe->mask_display;
+
+  /* does user want us to display a specific channel? */
```

Comment:
So now the exact same logic would have to be kept in sync between two places?
Maybe factor it out into helper function?